website/content/_index.md

```diff
@@ -45,13 +45,89 @@ Wash is distributed as a single binary, and the only prerequisite is [`libfuse`]
+
+This starts a small [Flask](http://flask.pocoo.org) webapp that keeps a count of how often it's been accessed in a [Redis](http://redis.io) instance that maintains state in a Docker volume.
+
+Navigate the filesystem to view running containers
+```
+$ wash
+Welcome to Wash!
+  Wash includes several built-in commands: wexec, find, list, meta, tail.
+  See commands run with wash via 'whistory', and logs with 'whistory <id>'.
+Try 'help'
+wash$ cd docker/containers
+wash$ list
+NAME                                                                CREATED               ACTIONS
+./                                                                  <unknown>             list
+45a0265546d63a8f1b0d17033748db1468dc49dfd09cdaf2db62c45a60e82aaf/   20 Mar 19 17:02 PDT   exec, list, metadata
+382776912d9373e6c4dc1350894b5290b22c36893a8fed08e2ba53fbb680c8a6/   20 Mar 19 17:02 PDT   exec, list, metadata
+wash$ list 382776912d9373e6c4dc1350894b5290b22c36893a8fed08e2ba53fbb680c8a6
+NAME            CREATED               ACTIONS
+./              20 Mar 19 17:02 PDT   exec, list, metadata
+metadata.json   <unknown>             read
+log             <unknown>             read, stream
+```
+
+Those containers are displayed as a directory, and provide access to their logs and metadata as files. Recent output from both can be accessed with common tools.
+```
+wash$ tail */log
+==> 382776912d9373e6c4dc1350894b5290b22c36893a8fed08e2ba53fbb680c8a6/log <==
+ * Serving Flask app "app" (lazy loading)
+ * Environment: production
+   WARNING: Do not use the development server in a production environment.
+   Use a production WSGI server instead.
+...
+
+==> 45a0265546d63a8f1b0d17033748db1468dc49dfd09cdaf2db62c45a60e82aaf/log <==
+1:C 21 Mar 2019 00:02:33.112 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+1:C 21 Mar 2019 00:02:33.112 # Redis version=5.0.4, bits=64, commit=00000000, modified=0, pid=1, just started
+1:C 21 Mar 2019 00:02:33.112 # Configuration loaded
+1:M 21 Mar 2019 00:02:33.113 * Running mode=standalone, port=6379.
+...
+```
+
+The list earlier also noted that the container "directories" support the *metadata* action. We can get structured metadata in ether YAML or JSON with `wash meta`
+```
+wash$ meta 382776912d9373e6c4dc1350894b5290b22c36893a8fed08e2ba53fbb680c8a6 -o yaml
+AppArmorProfile: ""
+Args:
+- app.py
+Config:
+...
+```
+
+We can interrogate the container more closely with `wexec`
+```
+wash$ wexec 45a0265546d63a8f1b0d17033748db1468dc49dfd09cdaf2db62c45a60e82aaf whoami
+root
+```
+
+Try exploring `docker/volumes` to interact with the volume created for Redis.
+
+### Record of Activity
+
+All operations have their activity recorded to journals in `wash/activity` under your user cache directory, identified by process ID and executable name. The user cache directory is `$XDG_CACHE_HOME` or `$HOME/.cache` on Unix systems, `$HOME/Library/Caches` on macOS, and `%LocalAppData%` on Windows.
 
 ## Current features
```

Comment:
I could use some concrete suggestions on how to improve the "current features" section.

Reply:
I haven't come up with anything better. With the example section before it, I think leaving the features section in may be ok.

Reply:
Yeah, I'm not sure either.